gulpfile.js

```diff
@@ -179,6 +187,8 @@ gulp.task('build', [
   'build/zone.js',
   'build/zone.js.d.ts',
   'build/zone.min.js',
+  'build/zone-error.js',
```

Comment:
Yes that is my thinking, but i am not sure we should do this just yet. Perhaps disabling things may be good enough for now?